openupgrade_scripts/scripts/base_automation/17.0.1.0/post-migration.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2024 Viindoo Technology Joint Stock Company (Viindoo)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from openupgradelib import openupgrade
+
+
+def _ir_act_server_update_name_if_base_automation(env):
+    act_servers = (
+        env["ir.actions.server"]
+        .with_context(active_test=False)
+        .search([("base_automation_id", "!=", False)])
+    )
+    if act_servers:
+        act_servers._compute_name()
```

Comment:
It is possible that certain models are not yet loaded in the migration and that an error occurs when executing this point.

```
2024-10-08 08:35:05,839 1 ERROR database OpenUpgrade: base_automation: error in migration script /opt/odoo/auto/addons/openupgrade_scripts/scripts/base_automation/17.0.1.0/post-migration.py: 'project.task' 
2024-10-08 08:35:05,844 1 ERROR database OpenUpgrade: 'project.task' 
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/openupgradelib/openupgrade.py", line 2298, in wrapped_function
    func(
  File "/opt/odoo/auto/addons/openupgrade_scripts/scripts/base_automation/17.0.1.0/post-migration.py", line 75, in migrate
    _ir_act_server_update_name_if_base_automation(env)
  File "/opt/odoo/auto/addons/openupgrade_scripts/scripts/base_automation/17.0.1.0/post-migration.py", line 13, in _ir_act_server_update_name_if_base_automation
    act_servers._compute_name()
  File "/opt/odoo/auto/addons/base_automation/models/ir_actions_server.py", line 47, in _compute_name
    action.name = f"{action_type} {action._stringify_path()}"
  File "/opt/odoo/custom/src/odoo/odoo/addons/base/models/ir_actions.py", line 679, in _stringify_path
    model = self.env[self.model_id.model]
  File "/opt/odoo/auto/addons/openupgrade_framework/odoo_patch/odoo/api.py", line 39, in __getitem__
    return Environment.__getitem__._original_method(self, model_name)
  File "/opt/odoo/custom/src/odoo/odoo/api.py", line 534, in __getitem__
    return self.registry[model_name](self, (), ())
  File "/opt/odoo/custom/src/odoo/odoo/modules/registry.py", line 213, in __getitem__
    return self.models[model_name]
KeyError: 'project.task'
```

Maybe move this part to an `end-migrate.py`?

Reply:
Yes, it can be move to end-migrate.
Unfortunately i'm no longer doing migration anymore, at least until next year. Feel free to cherry-pick my commit and create you own if you have to